packages/accounts/plugindefs/types.ts

```diff
@@ -1,10 +1,16 @@
-import type { Abi, Address, Chain } from "viem";
+import type { Abi, Address, Chain, Hex, parseAbiParameters } from "viem";
 
 export type PluginGenConfig = {
   abi: Abi;
   addresses: Record<number, Address>;
   chain: Chain;
   name: string;
-  // TODO: need to make this configurable to run in CI without requiring this
   rpcUrl?: string;
+  installConfig?: {
```

Comment:
What do y'all think about making this a requirement actually? I think that would simplify things a lot for consumers of plugins if the developers have to outline the install parameters

Reply:
I think this should be required.